config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'home' => 'static_pages#home'
```

Comment:
I thought these were 2 different things? One was defining the root path (root 'static_pages#home'). The other was creating the 'home_path' shortcut to avoid having to enter 'static_pages#home' throughout the code?